### PR TITLE
bug issue#223 resolved

### PR DESCRIPTION
**Pull request by ssaumyaa7**
**What kind of change does this PR introduce?**
-bugfix 

**Summary**
-There was bug while comparing, string and int cannot be === as there needs to be the same datatype but when we use == the datatypes are ignored rather the content is compared.

**Other information**
issue no. #223

### CODE REVIEW

File: resolvers/functions/creatorCheck.js
Comment (impiyush83) [inline resolvers/functions/creatorCheck.js:3]: == is loosely check. which is a bad practice
use === ,  match types of the variables

**Review by impiyush83 — CHANGES_REQUESTED** [review]
Use === instead of ==, by checking datatypes

**Comment by ssaumyaa7** [comment]
@impiyush83 taking your point in consideration I'll make the necessary changes again.... Thankyou

**Comment by ssaumyaa7** [comment]
@impiyush83 @palisadoes Rather than using == while comparing, converted type to string

**Review by impiyush83 — APPROVED** [review]

**Comment by palisadoes** [comment]
@ssaumyaa7  @impiyush83 Let's have more meaningful pull requests. For example using `===` consistently in all dart files would be a more useful, and require less interruptions for approvals. Please consider the time of the reviewers.

**Comment by ssaumyaa7** [comment]
@ranilwallace sir, @DMills27 sir, 
Please review, even after getting this merged. This is not reviewed hence not reflects my name in the contribution

**Pull request merged**
